packages/components/src/components/Table/Table.js

```diff
@@ -135,13 +135,24 @@ const Table = props => {
                 <TableBody>
                   {!loading && dataRows.length === 0 && (
                     <TableRow>
-                      <TableCell colSpan={headers.length}>
-                        <div className="noRows">
-                          {selectedNamespace === ALL_NAMESPACES
-                            ? emptyTextAllNamespaces
-                            : emptyTextSelectedNamespace}
-                        </div>
-                      </TableCell>
+                      {batchActionButtons.length > 0 && (
+                        <TableCell colSpan={headers.length + 1}>
+                          <div className="noRows">
+                            {selectedNamespace === ALL_NAMESPACES
+                              ? emptyTextAllNamespaces
+                              : emptyTextSelectedNamespace}
+                          </div>
+                        </TableCell>
+                      )}
+                      {batchActionButtons.length === 0 && (
```

Comment:
no need to duplicate this, the only difference is the length so let's just modify that:

```js
<TableCell colSpan={headers.length + (batchActionButtons.length ? 1 : 0)}>
  <div className="noRows">
    {selectedNamespace === ALL_NAMESPACES
      ? emptyTextAllNamespaces
      : emptyTextSelectedNamespace}
  </div>
</TableCell>
```

Reply:
ok, done that now :)